project/SparkBuild.scala

```diff
@@ -79,6 +83,11 @@ object BuildCommons {
   val testTempDir = s"$sparkHome/target/tmp"
 
   val javaVersion = settingKey[String]("source and target JVM version for javac and scalac")
+
+  // Google Protobuf version used for generating the protobuf.
+  val protoVersion = "3.21.0"
```

Comment:
why 3.21.0 used in sbt but 3.21.1 used in maven? is it possible to share the version variable across maven and sbt?